src/constants.jl

```diff
@@ -0,0 +1,89 @@
+module Const
```

Comment:
```suggestion
module Constants
```

There's no need for abbreviations?